content/docs/observability/_index.md

```diff
@@ -14,13 +14,14 @@ Description: >
 
 Metrics data is collected and pushed to the [OpenTelemetry Collector](https://github.com/open-telemetry/opentelemetry-collector), so it can be processed, and exported in a format consumable by Prometheus. SSL certificates for TLS between nodes are handled by [cert-manager](https://github.com/jetstack/cert-manager).
 
-CSM for Observability is composed of several services, each living in its own GitHub repository, that can be installed following one of the three deployments we support [here](deployment). Contributions can be made to this repository or any of the CSM for Observability repositories listed below. 
+CSM for Observability is composed of several services, each living in its own GitHub repository, that can be installed following one of the four deployments we support [here](deployment). Contributions can be made to this repository or any of the CSM for Observability repositories listed below. 
```

Comment:
Delete "below"

Reply:
```
{{<table "table table-striped table-bordered table-sm">}}
```
Probably github cannot render the HTML tag properly. Same for other branches,  e.g. [release-1.4-updated](https://github.com/dell/csm-docs/blob/release-1.4-updated/content/docs/observability/_index.md).
After publishing the doc using hugo, the HTML is correct, see http://10.225.6.31:1313/csm-docs/docs/observability/